.github/workflows/upload-s3.yml

```diff
@@ -0,0 +1,52 @@
+name: Extract metadata and upload to S3
+
+on:
+  schedule:
+    - cron: "0 2 * * *"
+  workflow_dispatch:
+
+jobs:
+  upload:
+    runs-on: ubuntu-latest
+
+    env:
+      AWS_REGION: us-east-2
+      S3_BUCKET: linc-dashboard
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.11
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+
+    - name: Install AWS CLI
+      run: |
+        curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+        unzip awscliv2.zip
+        sudo ./aws/install --update
+        aws --version
+        rm -rf awscliv2.zip aws
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v3
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: ${{ env.AWS_REGION }}
```

Comment:
I _think_ you might just only need this and not the step above where you install the CLI

https://medium.com/swlh/official-aws-for-github-actions-21b4cb39b8d3#:~:text=AWS%20created%20and%20support%20their,engineers%2C%20%E2%80%94%20doing%20God's%20work.

Reply:
Nice find.  I have removed the installation step.